pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[tool.black]
+line-length = 80
+
+[tool.isort]
+default_section = "THIRDPARTY"
+profile = "black"
+# Headings
+import_heading_stdlib = "Standard library"
+import_heading_thirdparty = "Third-party"
+import_heading_firstparty = "First-party"
+import_heading_localfolder = "Local"
+# Known modules to avoid misclassification
+known_standard_library = [
+    # Add standard library modules that may be misclassified by isort
+]
+known_third_party = [
+    # Add third-party modules that may be misclassified by isort
+    "wandb",
+]
+known_first_party = [
+    # Add first-party modules that may be misclassified by isort
+    "neural_lam",
+]
+
+[tool.flake8]
+max-line-length = 80
+ignore = [
+    "E203", # Allow whitespace before ':' (https://github.com/PyCQA/pycodestyle/issues/373)
+    "I002", # Don't check for isort configuration
+    "W503", # Allow line break before binary operator (PEP 8-compatible)
+]
+per-file-ignores = [
+    "__init__.py: F401", # Allow unused imports
+]
+
+[tool.codespell]
+skip = "requirements/*"
+
+# Pylint config
+[tool.pylint]
+ignore = [
+    "create_mesh.py", # Disable linting for now, as major rework is planned/expected
```

Comment:
do we need to open an issue for that to keep it in mind?

Reply:
The issue already exists upstream: https://github.com/joeloskarsson/neural-lam/issues/4